tests/test_auxtel_offset_ataos.py

```diff
@@ -0,0 +1,152 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import unittest
+import contextlib
+import types
+import logging
+import pytest
+
+from lsst.ts.idl.enums.Script import ScriptState
+from lsst.ts.standardscripts.auxtel import OffsetATAOS
+from lsst.ts.standardscripts import BaseScriptTestCase, get_scripts_dir
+
+from lsst.ts import salobj
+
+
+class TestOffsetATAOS(BaseScriptTestCase, unittest.IsolatedAsyncioTestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.log = logging.getLogger(type(cls).__name__)
+
+    async def basic_make_script(self, index):
+        self.script = OffsetATAOS(index=index, add_remotes=False)
+
+        return (self.script,)
+
+    @contextlib.asynccontextmanager
+    async def setup_mocks(self):
+        self.script.atcs.offset_aos_lut = unittest.mock.AsyncMock()
+        self.script.atcs.rem = types.SimpleNamespace(ataos=unittest.mock.AsyncMock())
```

Comment:
I think instead of replacing `self.script.atcs.rem` entirely with the namespace mock, you can probably just override `self.script.atcs.rem.ataos`,  e.g.:

```
self.script.atcs.rem.ataos = unittest.mock.AsyncMock()
```